frontend/src/components/elements/DataFrame/DataFrameUtil.tsx

```diff
@@ -122,14 +129,23 @@ export const getDimensions = (
     }
   }
 
+  const totalHeight = rows * rowHeight
```

Comment:
Maybe a brief comment here to describe what's going on here? eg

`// Allocate space for horizontal and vertical scrollbars, if needed`